src/main/java/io/dashboardhub/pipelinedashboard/config/ApplicationConfig.java

```diff
@@ -0,0 +1,14 @@
+package io.dashboardhub.pipelinedashboard.config;
```

Comment:
Application config is very generic, would it not be better to rename it to **DateConfig**?